webapi/core.py

```diff
@@ -80,7 +82,10 @@ def unpatch_web_ui(self):
 
     def render_patch(self, request):
 
-        if request.method != 'OPTIONS':
+        # twisted uses byte strings in deluge version 2.1.0 onward
+        use_bytes = version.parse(get_version()) >= version.parse('2.1.0')
```

Comment:
Good. One more thing: this line should rather go to a module level, to be evaluated once.

Reply:
I assume that by "module-level" you mean top-level in the file, done